### ISSUE TITLE
Crashing while trying to load world because of RFTools spawner

### ISSUE THREAD

**Issue by Xelat84** [FTBTeam/FTB-Presents-Stoneblock-2#687]
* **Modpack Version**: 1.16.1

* **Issue**: I've put pink slime syringe into RFTools spawner, and game crashed immediately, and after that world cannot be loaded. Most significant log record tells ```java.lang.IllegalStateException: The mob spawn amounts list for mob industrialforegoing:pink_slime (the default list) is the wrong size. Instead of 3 elements, it contained 0
    at mcjty.rftools.blocks.spawner.SpawnerTileEntity.getSpawnAmounts(SpawnerTileEntity.java:210)```

* **Link to Log or Crash File Paste**: https://paste.feed-the-beast.com/view/f05f824a or https://paste.dimdev.org/paketirelu.mccrash or https://pastebin.com/6ihJbfz3

* **Is it Repeatable?**: yes

* **Mod/s Affected**: RFTools 1.12-7.72, Industrial Foregoing 1.12.13 (1.12.2-1.12.2)

**Comment by TheLanceShipment** [comment]
Xelat84 did you find a fix? I'm having the same problem but with baby skeletons instead

**Comment by Xelat84** [comment]
> Xelat84 did you find a fix? I'm having the same problem but with baby skeletons instead

No, just edited the world save, and removed this block to keep playing.

BTW EnderIO spawners works fine.

**Comment by antonlam** [comment]
How to edit the world save or the way to remove the block?

**Comment by BokuNoMaxi** [comment]
I had the same problem with zombie_villager and fixed it. 
You don't have to delete the block, just update the config of rftools in "config/rftools/rftools.cfg"
In this file you will find entries looking like that:
```
S:"minecraft:zombie.spawnamount.0" <
        I
        minecraft:rotten_flesh
        0
        0.1
     >
    S:"minecraft:zombie.spawnamount.1" <
        B
        minecraft:dirt
        0
        0.2
     >
    S:"minecraft:zombie.spawnamount.2" <
        L
        
        0
        20.0
     >
```
then copy paste this code into the next line and edit it like this: 
```
S:"industrialforegoing:pink_slime.spawnamount.0" <
        I
        minecraft:rotten_flesh
        0
        0.1
     >
    S:"industrialforegoing:pink_slime.spawnamount.1" <
        B
        minecraft:dirt
        0
        0.2
     >
    S:"industrialforegoing:pink_slime.spawnamount.2" <
        L
        
        0
        20.0
     >
```
then you restart minecraft and it should load in.

Furthermore you could edit the line "minecraft:rotten_flesh" and "minecraft:dirt" to values that would make more sense to spawn pink slime for exemple: "minecraft:slimeball" and "minecraft:dyePink"
